src/main/kotlin/lotto/controller/LottoGame.kt

```diff
@@ -1,34 +1,62 @@
+import lotto.model.ManualLottoTicketGenerator
 import lotto.model.TicketQuantity
 import lotto.model.WinningCalculator
 import lotto.model.WinningStatistics
 import lotto.view.InputView
-import lotto.view.ResultView
+import lotto.view.OutputView
 
 class LottoGame {
-    private val resultView = ResultView()
+    private val outputView = OutputView()
     private val inputView = InputView()
 
     fun start() {
-        val ticketQuantity = TicketQuantity(InputView().getAmountOfMoney()).quantity
-        resultView.showQuantity(ticketQuantity)
-        val tickets = purchaseLottoTicket(ticketQuantity)
-        resultView.showLottoTicket(tickets)
-        val lottoTicket = LottoTicket(inputView.getWinnerNumber())
+        val automaticTicketQuantity = TicketQuantity(inputView.getAmountOfMoney()).quantity
+        val manualTicketQuantity = inputView.getManualQuantity().toInt()
+        val manualLottoTickets =
+            ManualLottoTicketGenerator(purchaseManualLottoTicket(manualTicketQuantity)).tickets
+        outputView.showQuantity(manualTicketQuantity, automaticTicketQuantity)
+        val automaticLottoTickets = purchaseAutomaticLottoTicket(automaticTicketQuantity)
+        val lottoTickets = joinTickets(manualLottoTickets, automaticLottoTickets)
+        outputView.showLottoTicket(lottoTickets)
+        val winnerTicket = LottoTicket(inputView.getWinnerNumber())
         val bonusNumber = LottoNumber(inputView.getBonusNumber())
-        val winningStatistics = drawWinnerNumber(tickets, lottoTicket, bonusNumber)
-        resultView.showWinningStatistics(winningStatistics)
+        val winningStatistics = drawWinnerNumber(lottoTickets, winnerTicket, bonusNumber)
+        outputView.showWinningStatistics(winningStatistics)
     }
 
-    fun purchaseLottoTicket(quantity: Int): List<RandomLottoTicketGenerator> {
-        return List(quantity) { RandomLottoTicketGenerator() }
+    private fun purchaseManualLottoTicket(quantity: Int): List<String> {
+        var result = mutableListOf<String>()
+        outputView.showManualNumber()
+        for (i in 1..quantity) {
+            result.add(inputView.getManualNumber())
+        }
+        return result
     }
 
-    private fun drawWinnerNumber(randomLottoTickets: List<RandomLottoTicketGenerator>, lottoTicket: LottoTicket, bonusNumber: LottoNumber): WinningStatistics {
-        return WinningCalculator(randomLottoTickets, lottoTicket, bonusNumber.value).winningStatistics
+    fun purchaseAutomaticLottoTicket(quantity: Int): List<LottoTicket> {
+        return AutomaticLottoTicketGenerator(quantity).tickets
+    }
+
+    private fun joinTickets(
+        manualLottoTickets: List<LottoTicket>,
+        automaticLottoTickets: List<LottoTicket>
+    ): List<LottoTicket> {
+        val lottoTickets = mutableListOf<LottoTicket>()
+        lottoTickets.addAll(manualLottoTickets)
+        lottoTickets.addAll(automaticLottoTickets)
+        return lottoTickets
```

Comment:
요건 별도로 joinTickets라는 메소드를 사용하지 않고 아래와 stdlib인 plus 혹은 + 연산자를 활용해보시면 좋을 것 같네요~

```suggestion
val lottoTickets = manualLottoTickets + automaticLottoTickets

// 혹은

val lottoTickets = manualLottoTickets.plus(automaticLottoTickets)
```